src/PushwooshMessage.php

```diff
@@ -24,6 +24,8 @@ class PushwooshMessage implements JsonSerializable
     protected $throughput;
     protected $url;
     protected $when;
+    protected $androidSilent = 0;
+    protected $iosSilent = 0;
```

Comment:
This way the keys will be only be included when set to `1`.
```suggestion
    protected $androidSilent = null;
    protected $iosSilent = null;
```

This is where the filtering happens:
https://github.com/laravel-notification-channels/pushwoosh/blob/f263fc621aff2bdef0b75c93c057c33692ec33d0/src/PushwooshMessage.php#L159-L161